hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -211,42 +205,22 @@ public static void writeDBCheckpointToStream(DBCheckpoint checkpoint,
       OutputStream destination)
```

Comment:
Let's remove this method, and replace the usage of DBCheckpointServlet.writeDBCheckpointToStream()  with HddsServerUtil.writeDBCheckpointToStream(). (There are three usages)